master/usage/troubleshooting/index.md

```diff
@@ -67,27 +81,50 @@ No IPv6 peers found.
 
 If you do not see this, please check the following.
 
-- Can your hosts ping each other?  There must be IP connectivity between the
-  hosts.
+- Make sure there is IP connectivity between your hosts.
+
+- Make sure your network allows the requisite BGP traffic on TCP port 179.
+
+### Disable Ubuntu (or GNOME) NetworkManager
+
+Disable [NetworkManager](https://help.ubuntu.com/community/NetworkManager) before
+attempting to use {{site.prodname}} networking.
+
+NetworkManager manipulates the routing table for interfaces in the default network
+namespace where {{site.prodname}} veth pairs are anchored for connections to containers.
+This can interfere with the {{site.prodname}} agent's ability to route correctly.
+
+You can configure interfaces in the `/etc/network/interfaces` file if the
+NetworkManager removes your host's interfaces. See the Debian
+[NetworkConfiguration](https://wiki.debian.org/NetworkConfiguration)
+guide for more information.
+
+## Running `sudo calicoctl ...` with Environment Variables
```

Comment:
while you're in here, if you could possibly remove the backticks from the sudo calicoctl from the heading, that would be amazing